app/views/episodes/edit.slim

```diff
@@ -0,0 +1,73 @@
+.container
+  .py-3.text-center
+    img.d-block.mx-auto.mb-4[src="https://i1.sndcdn.com/avatars-000591178137-3dyxzr-t200x200.jpg" alt="proConf logo" width="72" height="72"]
```

Comment:
don't use static links, you can upload this image from 'public' like `/proconf_logo.jpg`